### PR TITLE
Marshmallows for breakfast

### PR DESCRIPTION
**Pull request by showeirh**

### CODE REVIEW

**Comment by showeirh** [comment]
my story

**Comment by showeirh** [comment]
my story

**Comment by showeirh** [comment]
Please review my story, @casey-collab

File: english/breakfast/breakfast.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+You see a pot of coffee near you and you realize that you are hungry. Well, you poured coffee in a mug and picked some
```
Comment (casey-collab) [inline english/breakfast/breakfast.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by showeirh** [comment]
please take a second look

**Comment by showeirh** [comment]
Please take a second look.

**Comment by cbuckey-uda** [comment]
@casey-collab

File: english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
```
Comment (casey-collab) [inline english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/marshmallow.md [hunk 3]
```diff
@@ -274,3 +274,4 @@ Do you:
 [Eat the Marshmallow](eatmarshmallow/eatmarshmallow.md)
 
 [Escape into Willy Wonka Lego-World](lego-world/lego-world.md)
+[Eat breakfast](breakfast/breakfast.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md (hunk 2 above)
Comment (casey-collab) [inline english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/breakfast/breakfast.md (hunk 1 above)
Comment (cbuckey-uda) [inline english/breakfast/breakfast.md]: I believe this comment is old and you can ignore it.

File: english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md (hunk 2 above)
Comment (cbuckey-uda) [inline english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md]: It looks to me like this was a previous attempt to add a link to `breakfast.md`, before you realized that you needed to add the link to an existing file, such as `marshallow.md`. Is that accurate? If so, and you no longer want this page, you can remove it using the command `git rm`.

Comment (showeirh) [inline english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md]: You are right. This link [get breakfast] (../breakfast/breakfast.md) was my first attempt to link to my story. But when Casey said it's broken, I replaced it with [get breakfast](../english/breakfast/breakfast.md) which fixed the broken link problem.
I tested the new link which lives in Udacity's marshmallow.md file and I was able to reach my story.
I hope it's ok for merge now.

Thanks for you real-time collaboration.  


     On Tuesday, June 9, 2015 6:33 AM, Caroline Buckey <notifications@github.com> wrote:
   

 In english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md:> @@ -0,0 +1 @@
> +Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
It looks to me like this was a previous attempt to add a link to breakfast.md, before you realized that you needed to add the link to an existing file, such as marshallow.md. Is that accurate? If so, and you no longer want this page, you can remove it using the command git rm.—

Comment (cbuckey-uda) [inline english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md]: My point is that I can still see the old link - `../breakfast/breakfast.md`. I can see both this link, and your fixed link, so I believe you need to remove this one.

Comment (showeirh) [inline english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md]: I've given up.I really don't know what to do other than deleting the old link and replaced it with the new one that is reachablefrom the change branch in the marshmallow.md file. 

Your intensive care is needed.
Thanks'Hisham Showeir
 


     On Wednesday, June 10, 2015 2:35 AM, Caroline Buckey <notifications@github.com> wrote:
   

 In english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md:> @@ -0,0 +1 @@
> +Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
My point is that I can still see the old link - ../breakfast/breakfast.md. I can see both this link, and your fixed link, so I believe you need to remove this one.—

Comment (showeirh) [inline english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md]: I forgot to tell you I tried git rm command which was not of help.
 


     On Wednesday, June 10, 2015 5:50 AM, Hisham Showeir <showeirh@rocketmail.com> wrote:
   

 I've given up.I really don't know what to do other than deleting the old link and replaced it with the new one that is reachablefrom the change branch in the marshmallow.md file. 

Your intensive care is needed.
Thanks'Hisham Showeir
 


     On Wednesday, June 10, 2015 2:35 AM, Caroline Buckey <notifications@github.com> wrote:
   

 In english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md:> @@ -0,0 +1 @@
> +Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
My point is that I can still see the old link - ../breakfast/breakfast.md. I can see both this link, and your fixed link, so I believe you need to remove this one.—

**Comment by showeirh** [comment]
I believe I removed the old link using the command "git rm  english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md".
Please take another look.

File: english/marshmallow.md [hunk 4]
```diff
@@ -280,7 +280,14 @@ Do you:
...
+
+[Eat breakfast] (../english/breakfast/breakfast.md)
+
+||||||| merged common ancestors
+=======
 
 [The inception protocol](dream_in_dream/inception.md)
 
 [Pull out your proton pack!](ghost-buster/ghost-busted.md)
+>>>>>>> 1aead4324b63c9802e63bd83910671bd25627047
```
Comment (cbuckey-uda) [inline english/marshmallow.md]: It looks like this merge conflict didn't get resolved properly - the special merge conflict lines merge conflict will need to be removed.

**Comment by cbuckey-uda** [comment]
Thanks, it looks like using `git rm` to remove the old link worked! I'm glad you figured it out. Now if you remove the merge conflict lines, I believe this should be good.

**Comment by showeirh** [comment]
merge conflict resolved@casey-collab

**Comment by casey-collab** [comment]
It looks like this pull request deletes quite a few lines. Was this a mistake?

**Comment by showeirh** [comment]
No, extra spaces were deleted to unify spacing across the file.

**Comment by cbuckey-uda** [comment]
Looks good to me. Thanks for fixing up the spacing. :egg: :coffee:

**Comment by showeirh** [comment]
Thanks Caroline for walking me throughout the merge process.P.S. You are good. You were up-to-date with every step that I took and was informative.
Sincerely,Hisham Showeir. 

     On Friday, June 12, 2015 8:09 PM, Caroline Buckey <notifications@github.com> wrote:
   

 Merged #1543.—

**Comment by cbuckey-uda** [comment]
You're welcome! I'm glad I could help.

**Pull request merged**